data_preprocessing/collectivite_territoriale.py

```diff
@@ -189,51 +188,3 @@ def process_elus_files(url, colname):
         }
     )
     return df_elus
-
-
-def generate_updates_colter(df_colter, current_color):
-    for index, row in df_colter.iterrows():
-        colter_code_insee = row["colter_code_insee"]
-        if row["colter_code_insee"] != row["colter_code_insee"]:
-            colter_code_insee = None
-        yield {
-            "_op_type": "update",
-            "_index": "siren-" + current_color,
-            "_type": "_doc",
-            "_id": row["siren"],
-            "doc": {
-                "colter_code_insee": colter_code_insee,
-                "colter_code": row["colter_code"],
-                "colter_niveau": row["colter_niveau"],
-            },
-        }
-
-
-def generate_updates_elu(df_elus, current_color):
```

Comment:
bye awesome functions

Reply:
ahahaha all good things must come to en end